docs/05_components/doc-management/doc-management-server/02_server-endpoints.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: 'Document Management - Endpoints'
+sidebar_label: 'Endpoints'
+id: doc-management-endpoints
+keywords: [doc-management, endpoints]
+tags:
+  - doc-management
+  - endpoints
+---
```

Comment:
This is a bit scant. Can we also add information on configuring the file storage manager, different options and relevant sysdef keys for s3, sharepoint and local?